js/state/syncUtil.js

```diff
@@ -303,16 +304,30 @@ module.exports.getObjectById = (objectId, category, appState) => {
  * Given an bookmark folder objectId, find the folder and return its folderId.
  * @param {Immutable.List} objectId
  * @param {Immutable.Map=} appState
+ * @param {Immutable.List=} records
```

Comment:
done in https://github.com/brave/browser-laptop/pull/8088/files#diff-dbafb8aac78fc0a84a0dcc51f799da7cR64